docs/sources/articles/systemd.md

```diff
@@ -24,14 +24,38 @@ If you want Docker to start at boot, you should also:
 ## Custom Docker daemon options
 
 There are a number of ways to configure the daemon flags and environment variables
-for your Docker daemon. 
+for your Docker daemon.
 
 If the `docker.service` file is set to use an `EnvironmentFile`
 (often pointing to `/etc/sysconfig/docker`) then you can modify the
-referenced file.
+referenced file. To check if this is the case:
 
-Or, you may need to edit the `docker.service` file, which can be in
-`/usr/lib/systemd/system`, `/etc/systemd/service`, or `/lib/systemd/system`.
+    $ sudo systemctl show docker --property EnvironmentFile
+    EnvironmentFile=-/etc/sysconfig/docker
+
+Otherwise, you can see the location and content of the service files:
+
+    $ sudo systemctl cat docker
+    # /lib/systemd/system/docker.service
+    [Unit]
+    Description=Docker Application Container Engine
+    Documentation=http://docs.docker.com
+    After=network.target docker.socket
+    Requires=docker.socket
+
+    [Service]
+    ExecStart=/usr/bin/docker -d -H fd://
+    MountFlags=slave
+    LimitNOFILE=1048576
+    LimitNPROC=1048576
+    LimitCORE=infinity
+
+    [Install]
+    WantedBy=multi-user.target
+
+The files located in `/usr/lib/systemd/system` or `/lib/systemd/system` should not
+be edited. Instead, you should override the properties you need to change by
+following [the second example](#http-proxy).
```

Comment:
I'm thinking of moving the instructions for customizing first, and the 'warning' after that.
Also more explicitly refering to "which" example we're refering to.

Something like:

```Markdown
You can customize the Docker daemon options using override files as explained in the
[HTTP Proxy example](#http-proxy) below. The files located in `/usr/lib/systemd/system` 
or `/lib/systemd/system` contain the default options and should not be edited.
```

Perhaps @moxiegirl has a better suggestion